scripts/frontend/dev-server.js

```diff
@@ -13,8 +13,11 @@ function buildUrl(req) {
     const DEFAULT_URL =
         'https://www.theguardian.com/money/2017/mar/10/ministers-to-criminalise-use-of-ticket-tout-harvesting-software';
     const url = new URL(req.query.url || DEFAULT_URL);
+    // searchParams will only work for the first set of query params because 'url' is already a query param itself
+    const searchparams = url.searchParams && url.searchParams.toString();
```

Comment:
I think `.searchParams` will _always_ be truthy, even if there is no query string:

```js
new URL("https://www.theguardian.com").searchParams
=> URLSearchParams {}
```

And calling `toString()` on it in this case will give back an empty string, so it's probably safe to remove the `url.searchParams &&` here (or perhaps use a different mechanism to decide if there are any query string args).

Reply:
Ah, yes. I was doing something different before when I added this check. Removed now. 👍